client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -146,6 +144,11 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             })
     }
 
+    const canChangeStatusNewToInProcess = (investigationStatus: Number, investigationInvestigator? : string) => {
+        const investigatorTocheck = investigationInvestigator || currInvestigatorId;
+        return investigationStatus === InvestigationMainStatusCodes.NEW && (user.userType === userType.INVESTIGATOR || investigatorTocheck === user.id);
```

Comment:
Is this check correct? Shouldn’t be if it is an admin and not investigator?

Reply:
No.  Only Investigator or if that investigation is associated to that Admin. Only than the investogation's status automatic turns to InProcess